classes/itembase.php

```diff
@@ -1537,23 +1537,51 @@ public function userform_add_disabledif($mform) {
     /**
      * Starting from the info stored into $answer, this function returns the corresponding content for the export file.
```

Comment:
You should amend this method docs, because t's the same than the `userform_db_to_export()` one, sort of "Provide information for some common answers used by all items"